src/index.html

```diff
@@ -5,8 +5,415 @@
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
+  
   <body>
-    <h1>Catalog</h1>
+    <div class="header">
+      <header class="header__style">
+        <a class="header__logo" href="#home">
+          <img 
+            src="images/logo.png"
+            alt="Moyo Logo"
+          >
+        </a>
+
+        <nav class="nav">
+          <ul class="nav__list">
+            <li>
+              <a class="nav__link is-active" href="#apple">
+                apple
+              </a>
+            </li>
+
+            <li class="nav__item">
+              <a class="nav__link" href="#samsung">
+                samsung
+              </a>
+            </li>
+
+            <li class="nav__item">
+              <a class="nav__link" href="#smartphones">
+                smartphones
+              </a>
+            </li>
+
+            <li class="nav__item">
+              <a
+                class="nav__link"
+                data-qa="nav-hover"
+                href="#laptops&computers"
+              >
+                laptops & computers
+              </a>
+            </li>
+
+            <li class="nav__item">
+              <a class="nav__link" href="#gadgets">
+                gadgets
+              </a>
+            </li>
+
+            <li class="nav__item">
+              <a class="nav__link" href="#tablets">
+                tablets
+              </a>
+            </li>
+
+            <li class="nav__item">
+              <a class="nav__link" href="#photo">
+                photo
+              </a>
+            </li>
+
+            <li class="nav__item">
+              <a class="nav__link" href="#video">
+                video
+              </a>
+            </li>
+          </ul>
+        </nav>
+      </header>
+    </div>
+
+    <main class="card">
+      <div class="card__form" data-qa="card">
+        <div class="card__image">
+        </div>
+
+        <h2 class="card__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h2>
+
+        <p class="card__code">
+        Product code: 195434
+        </p>
+        
+        <div class="card__rate">
+          <div class="card__stars">
+            <div class="card__star"></div>
+            <div class="card__star"></div>
+            <div class="card__star"></div>
+            <div class="card__star"></div>
+            <div class="card__star"></div>
+          </div>
+          <div class="card__review">Reviews: 5</div>
+        </div>
+
+        <div class="card__price">
+          <p class="card__tag">Price:</p>
+          <p class="card__amount">$2,199</p>
+        </div>
+
+        <form 
+          data-qa="card-hover"
+          class="card__button"
+          action="#buy"
+        >
+          <button
+            class="card__submit"
+            type="submit"
+          >
+          buy
+          </button>
+        </form>
+      </div>
+
+      <div class="card__form">
```

Comment:
why do you need this wrapper

Reply:
i'm making gap between each card, to see where it starts and where is over